### DISCUSSION TITLE
InstallExecuteSequence not work  when processing some on C:\Program Files

### DISCUSSION THREAD

**Discussion by alicera** [wixtoolset/issues#6990] [Questions]
I write a InstallExecuteSequence and use it to delete the folder  aa that I install with installer.
It cant not work. (I have add some file on folder aa) 

By the way, if I assign the path to  C:\myfolder\aa
it can work
```
<Binary Id="CommandPrompt" SourceFile="C:\Windows\System32\cmd.exe" />
<CustomAction Id="DeleteFolder" BinaryKey="CommandPrompt" 
      ExeCommand='/c rmdir /s /q "C:\Program Files\aa\"' 
      Execute="immediate" Return="ignore" Impersonate="no" />

<InstallExecuteSequence>
<Custom Action="DeleteFolder" After='RemoveFolders'>Installed</Custom>
</InstallExecuteSequence>
```

**Comment by robmen** [comment]
Ending a command line with `\"` can create issues with cmd.exe string parsing. You need to escape the value properly.
